include/imu_ros2/setting_declarations.h

```diff
@@ -30,4 +30,7 @@ enum IIODeviceName {
     ADIS1657X
 };
 
+#define PRINT_ACCELGYRO 0
+#define PRINT_VELANG 1
```

Comment:
I would also rename these into something related to the name of the parameter. something like:
ACCEL_GYRO_BUFFERED_DATA
DELTAVEL_DELTAANGL_BUFFERED_DATA